test/test.hs

```diff
@@ -2342,6 +2710,16 @@ instance Arbitrary (Expr Buf) where
 instance Arbitrary (Expr End) where
   arbitrary = sized genEnd
 
+data OpContract = OpContract [Op]
```

Comment:
This is also defined in Debug.hs. Is this intentional?

Reply:
Oh, yeah, that was wrong, thanks! I used it to debug. I removed it from `Debug.hs` now :)